website/content/docs/enterprise/hsm/behavior.mdx

```diff
@@ -15,14 +15,14 @@ effect when using Vault with an HSM.
 
 ## Key split between unseal keys and recovery keys
 
-Normally, Vault uses a single set of unseal keys to perform both decryption of
-the cryptographic barrier and to authorize recovery operations, such as the
+Normally, Vault uses a single set of unseal keys to both decrypt the
+cryptographic barrier and to authorize recovery operations, such as the
 [`generate-root`](/vault/api-docs/system/generate-root)
 functionality.
 
-When using an HSM, because the HSM automatically unseals the barrier but
-recovery operations should still have human oversight, Vault instead uses two
-sets of keys: unseal keys and recovery keys. 
+When using an HSM, Vault instead uses two sets of keys: unseal keys and recovery
+keys. This lets the HSM automatically unseal the barrier but still requires a
+human to oversee recovery operations.
```

Comment:
I see how troublesome the original text was, and a bit confusing.

How about:

```suggestion
When using an HSM, the HSM automatically unseals the Vault barrier, but recovery
operations still require human oversight. To authorize recovery operations,
Vault generates recovery keys.
```

Reply:
I think the point here was to highlight to the user that they will have two sets of keys instead of one. Are you saying that the HSM doesn't use a key to unseal the Vault barrier?

Reply:
I'm just guessing that that was what we were trying to tell people based on the section heading, but maybe that's not the most important point?

Reply:
Yes, that's much clearer.